MiqoCraftCore/MiqoCraftCore.cs

```diff
@@ -287,9 +287,22 @@ public static string GenerateScenario(List<FFXIVItem> iItemToGenerate, MiqobotSc
             string allPreset = "";
 
             List<string> catalysts = GetCatalysts();
-            if (!iOptions.IgnoreCatalysts) catalysts.Clear();
-
+            List<string> Namedcatalysts = GetCatalysts();
```

Comment:
Not the best solution, but it works. I was trying to use your list, but somehow you erased in somewhere so i can do a string compare with nodeName to check if is shard/crystal/cluster and enforce a special rotation for it